fuel-core-interfaces/src/db.rs

```diff
@@ -180,15 +181,13 @@ pub mod helpers {
                 receipts_root: Default::default(),
                 script,
                 script_data: vec![],
-                inputs: vec![Input::Coin {
+                inputs: vec![Input::CoinSigned {
```

Comment:
We could consider including a predicate too, for test purposes

Reply:
yeah i'll do that in the future, hence why predicates are under a feature flag for now as unstable